src/infrastructure/persistance/item_api_v1_repository.py

```diff
@@ -41,12 +42,12 @@ def post(self, item: dict) -> Item:
         )
         return Item(**response.json()["result"])
 
-    def patch(self, id: int, item: dict) -> dict:
+    def patch(self, id: int, patch_info: PatchInfo) -> PatchInfo:
         response = requests.patch(
             f"http://api.rollbar.com/api/1/item/{id}",
-            data=json.dumps(item),
+            data=json.dumps(dict(patch_info)),
             headers={
                 "X-Rollbar-Access-Token": os.environ.get("ROLLBAR_WRITE_TOKEN", "")
             },
         )
-        return dict(response.json()["result"])
+        return PatchInfo(**response.json()["result"])
```

Comment:
Can we return an `Item` instead a **dto**?

Reply:
if the patch requests goes right it returns the same info you passed as the response